bigtable/client/client_options_test.cc

```diff
@@ -46,3 +46,71 @@ TEST(ClientOptionsTest, EditCredentials) {
   EXPECT_EQ(typeid(grpc::InsecureChannelCredentials()),
             typeid(client_options_object.credentials()));
 }
+
+TEST(ClientOptionsTest, SetGrpclbFallbackTimeout) {
+  bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
+  client_options_object.SetGrpclbFallbackTimeout(5);
+  grpc::ChannelArguments c_args = client_options_object.channel_arguments();
+  grpc_channel_args test_args = c_args.c_channel_args();
+  EXPECT_EQ((size_t)2, test_args.num_args);
```

Comment:
This is super hard to understand.  Is there a more readable accessor?  Or somehow a way to test that I modified the `FallbackTimeout` and not the `CompressionAlgorithm`

Reply:
Added a check for Channel Argument type.